package.json

```diff
@@ -35,13 +36,24 @@
     "chai": "^4.2.0",
     "eslint": "^7.7.0",
     "eslint-config-eslint": "^6.0.0",
+    "eslint-plugin-jsdoc": "^22.1.0",
+    "eslint-plugin-node": "^11.1.0",
     "eslint-release": "^3.1.0",
     "mocha": "^8.1.1"
   },
   "dependencies": {
     "ajv": "^6.12.4",
     "debug": "^4.1.1",
+    "espree": "^7.3.0",
+    "globals": "^12.1.0",
+    "ignore": "^4.0.6",
     "import-fresh": "^3.2.1",
+    "js-yaml": "^3.13.1",
```

Comment:
Question: Does moving these dependencies over here allow us to remove any from ESLint?

Reply:
Probably, we could try with deleting some and see if `node/no-extraneous-require` warns.

Reply:
I wouldn't remove dependencies from ESLint at this point. I'm still trying to get this package working correctly. :)